lambdas/question/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionHandler.java

```diff
@@ -57,6 +59,7 @@ public class QuestionHandler
     public static final String HEADER_SESSION_ID = "session-id";
     public static final String LAMBDA_NAME = "get_question";
     public static final String ERROR_KEY = "error";
+    public static final String VERIFICATION_SCORE_PARAM_NAME = "di-ipv-cri-kbv-api/strengthScore";
```

Comment:
`di-ipv-cri-kbv-api/strengthScore` is currently strengthSore see https://github.com/govuk-one-login/architecture/blob/83e2d11898edcd82edaccf17bc29ce6c403b5fbf/rfc/0024-identity-vc-for-checks.md#evidence-properties